moped-editor/src/components/GridTable/GridTable.js

```diff
@@ -507,10 +490,13 @@ const GridTable = ({ title, query, searchTerm }) => {
                                       <>
                                         {query.config.columns[
                                           column
-                                        ].hasOwnProperty("chip")
-                                          ? buildChip(
+                                        ].hasOwnProperty("badge")
```

Comment:
can this ☝️ be:
```javascript
 query.config.columns[column]?.badge
```
?

Reply:
This is old code from vision zero, iirc. I believe hasOwnProperty is there because we need a True/False return, so it is only matter of making it a truthy statement:

<img width="241" alt="2021-12-08_17-07-28" src="https://user-images.githubusercontent.com/5282430/145305577-94756bf7-2edc-46b8-a6a1-185e4b6d85e8.png">